frontend/src/my_team/MyTeamView.jsx

```diff
@@ -56,45 +44,46 @@ function MyTeamView() {
     queryUrl.searchParams.append("userId", userContext.user.id);
     const response = await fetch(queryUrl);
     const results = await response.json();
+    setRecommendations(results);
   }
 
-  useEffect(() => {
-    fetchTeamPlayers("myTeamPlayers");
-    fetchTeamPlayers("opponents");
-  }, []);
-
-  useEffect(() => {
-    fetchPlayersStats();
-  }, [myTeamPlayers]);
+  //TODO: implement better UI feedback on recommending ideal players
 
   return (
     <div className="view myTeamView">
       <Header />
-      <div className="playerCardList">
-        {/* TODO: Add scrolling styling or move to separate div if more team stats added */}
+      {myTeamPlayers.isPending && <AppLoader />}
+      {myTeamPlayers.data && (
+        <div className="playerCardList">
+          {myTeamPlayers.data.map((player) => {
+            return (
+              <PlayerCard
+                key={player.id}
+                player={player}
+                teamPlayers={myTeamPlayers.data}
+                userId={userContext.user.id}
+                teamType="myTeamPlayers"
+                refetchPlayers={myTeamPlayers.refetch}
+              />
+            );
+          })}
+        </div>
```

Comment:
is the desired behavior to show loader above the data when it is loading? or should it be hidden

Reply:
Yeah, the loader is meant to display while the data hasn't been loaded yet